docs/site/src/docs/advanced/kyverno-build-in.md

```diff
@@ -0,0 +1,86 @@
+# Kyverno build-in
+
+## Motivations
+
+It's common that some k8s clusters have their own private image registry,and they don't want to pull images from other registry for safety reasons.this page is about how to introduce kyverno in k8s cluster,which will redirect image pull request to Specified registry.
+
+## Uses case
+
+### step 1：choose a base image
+
+choose a base image which can create a k8s cluster with at least one master node and one work node.To demonstrate the workflow,I will use `kubernetes:v1.19.8`.you can get the same image by executing `sealer pull kubernetes:v1.19.8`.
```

Comment:
Kyverno is for official docker image, if using sealer-docker, we needn't use kyverno. So the BaseImage should not be kubernetes:v1.19.8